src/Stratis.SmartContracts.CLR/SCL/ECRecover.cs

```diff
@@ -34,20 +41,11 @@ private static Address[] VerifySignatures(string[] signatures, byte[] message, A
         /// </summary>
         /// <param name="signatures">The list of signatures of the message.</param>
         /// <param name="message">The message that was signed.</param>
-        /// <param name="addresses">The addresses returned are intersected with these addresses.</param>
-        /// <param name="verifiedAddresses">The list of addresses that produced the signatures and constrained to the list provided in <paramref name="addresses"/>.</param>
-        /// <returns>The boolean value returned only indicates whether the operation could be performed. The number of verified addresses should still be checked.</returns>
-        public static bool TryVerifySignatures(string[] signatures, byte[] message, Address[] addresses, out Address[] verifiedAddresses)
+        /// <param name="addresses">The addresses returned are intersected with these addresses (if not <c>null</c>).</param>
+        /// <returns>The list of addresses that produced the signatures and constrained to the list provided in <paramref name="addresses"/>.</returns>
+        public static Address[] GetVerifiedSignatures(string[] signatures, string message, Address[] addresses)
```

Comment:
This is an API change to the SCL library. I don't think these should be made randomly in PRs like this but rather as part of a structured API design for the initial SCL release.